src/Keboola/StorageApi/Options/BackendConfiguration.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageApi\Options;
+
+class BackendConfiguration
+{
+    private string $context;
+
+    private string $size;
+
+    public function __construct(
+        string $context,
+        string $size
```

Comment:
Proč je to povinné?
Co budu posílat v `size` když nebude nastavená (synapse žádnou size nemá)
Nebo co budu posílat `context` v budoucnu když to budeme implementovat pro ❄️ a budeme znát jen size?

Reply:
Enom sem to tak nahodil, spíš tomu udělám fluent setters

Reply:
dal sem tomu fluent interface